state-chain/pallets/cf-ingress-egress/src/boost_pool.rs

```diff
@@ -240,7 +250,12 @@ where
 			(provided_amount, fee)
 		};
 
-		self.use_funds_for_boosting(prewitnessed_deposit_id, provided_amount, fee_amount)?;
+		// NOTE: only a portion of the boost fee goes to the boost pool,
```

Comment:
nit: the comment reads the other way around to the code. in the code below we're taking the network fee first